.eslintrc.json

```diff
@@ -47,15 +43,10 @@
     "react/jsx-filename-extension": [2, { "extensions": [".js", ".jsx", ".ts", ".tsx"] }],
     "@typescript-eslint/ban-ts-comment": ["error", { "ts-ignore": "allow-with-description" }],
     "no-shadow": "off",
-    "@typescript-eslint/no-shadow": ["error"]
+    "@typescript-eslint/no-shadow": ["error"],
+    "@typescript-eslint/no-unused-expressions": "warn"
```

Comment:
you can decide if this should be an error or not. I change the pattern from `a && b` to `if(a) b` because of it

Reply:
Good.
allowShortCircuit allows  a && b.
allowTernary allows a ? b : c.
I will make both be true